content/docs/2.17/scalers/temporal.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Temporal"
+availability = "v2.17+"
+maintainer = "Community"
+description = "Scale applications based on Temporal task queue."
+go_file = "temporal_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `temporal` trigger that scales based on a temporal task queue.
+
+```yaml
+triggers:
+  - type: temporal
+    metadata:
```

Comment:
please add here all parameters that are supported, the optional ones could have comment: `# Optional`